libs/core/src/lib/flexible-column-layout/constants.ts

```diff
@@ -0,0 +1,23 @@
+  
+export const ONE_COLUMN_START_FULL_SCREEN = 'OneColumnStartFullScreen';
+export const ONE_COLUMN_MID_FULL_SCREEN = 'OneColumnMidFullScreen';
+export const ONE_COLUMN_END_FULL_SCREEN = 'OneColumnEndFullScreen';
+export const TWO_COLUMNS_START_EXPANDED = 'TwoColumnsStartExpanded';
+export const TWO_COLUMNS_MID_EXPANDED = 'TwoColumnsMidExpanded';
+export const TWO_COLUMNS_END_EXPANDED = 'TwoColumnsEndExpanded';
+export const THREE_COLUMNS_MID_EXPANDED = 'ThreeColumnsMidExpanded';
+export const THREE_COLUMNS_END_EXPANDED = 'ThreeColumnsEndExpanded';
+export const THREE_COLUMNS_START_MINIMIZED = 'ThreeColumnsStartMinimized';
+export const THREE_COLUMNS_END_MINIMIZED = 'ThreeColumnsEndMinimized';
+
+export const SM_SCREEN_SIZE = 'sm';
+export const MD_SCREEN_SIZE = 'md';
+export const LG_SCREEN_SIZE = 'lg';
+export const XL_SCREEN_SIZE = 'xl';
+
+export type ScreenSize = 'sm' | 'md' | 'lg' | 'xl';
+
+export type Layout = 'OneColumnStartFullScreen' | 
+'OneColumnMidFullScreen' | 'OneColumnEndFullScreen' | 'TwoColumnsStartExpanded' | 'TwoColumnsMidExpanded' | 'TwoColumnsEndExpanded' | 'ThreeColumnsMidExpanded' | 'ThreeColumnsEndExpanded' | 'ThreeColumnsStartMinimized' | 'ThreeColumnsEndMinimized';
```

Comment:
What if we use `enum` instead of `type Layout` and `const`s that created above?

Reply:
I would stick to `type`, but with changed named to `FlexibleColumnLayout`